docs/user-guide/concepts.md

```diff
@@ -1,5 +1,7 @@
 # Concepts
 
+![architecture](../public/architecture-2.png)
+
 ## Components
```

Comment:
Could you please change the content order in components to:
* Meta
* Frontend
* Datanodes

that is the same as the image.